src/components/NotFound.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+
+export function NotFound() {
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+        justifyContent: "center",
+        height: "100%"
+      }}
+    >
+      <Typography variant="h1" sx={{ fontSize: "18px", color: "#333" }}>
```

Comment:
Not sure if now is the time to think about this, but do we have some global theme somewhere where these colors are stored? In the future we may want to support theming (so that we can integrate nicely with JupyterLab, among other reasons).

Reply:
Yes, there is a [theme file](https://github.com/conda-incubator/conda-store-ui/blob/8d4785c654db349cde4bcb75fffb9f3579efaa6b/src/theme.tsx) but it's probably incomplete and there hasn't been discipline in using it throughout the codebase. 

But it good to get in the habit, so I updated the color values to use "secondary.dark" instead of "#333".